src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
```

Comment:
There should be an empty line between the closing link tag and the closing head tag to separate multiline sibling blocks of HTML.